frontend/components/Course/StudentQueuePage/QuestionForm.tsx

```diff
@@ -1,14 +1,10 @@
 import React, { useState } from "react";
 import { Segment, Form, Header, Button } from "semantic-ui-react";
+import { mutateResourceListFunction } from "@pennlabs/rest-hooks/dist/types";
```

Comment:
@davish Should we export types from the library? Or did you choose not to export this for good reason?

Reply:
yeah, we should. I wish there was a way for us to alias this to `rest-hooks/types` but I'll put it in the index exports in the next release (wow that sounds so professional)